tests/erica_legacy/elster_xml/grundsteuer/test_elster_data_representation.py

```diff
@@ -49,16 +53,68 @@ def test_if_two_persons_then_attributes_set_correctly(self):
         person2["persoenlicheAngaben"]["vorname"] = "Rubeus"
         eigentuemer_obj = Eigentuemer.parse_obj(
             {"person": [person1, person2], "verheiratet": {"are_verheiratet": False}})
+        grundstueck_obj = SampleGrundstueck().parse()
 
-        result = EGW1(eigentuemer_obj)
+        result = EGW1(eigentuemer_obj, grundstueck_obj)
 
         assert result.Ang_Feststellung == EAngFeststellung()
         assert len(result.Eigentuemer) == 2
         assert result.Eigentuemer[0] == EPersonData(Person.parse_obj(person1), person_index=0)
         assert result.Eigentuemer[1] == EPersonData(Person.parse_obj(person2), person_index=1)
         assert result.Eigentumsverh == EEigentumsverh(eigentuemer_obj)
         assert result.Empfangsv is None
-        assert len(vars(result)) == 4
+        assert len(vars(result)) == 7
```

Comment:
Wherever you adapted the len() check, we should also include the newly added fields to the test before, I think.

Reply:
Reworked all tests.